lib/widgets/upload/find_ad_details_fom.dart

```diff
@@ -33,14 +35,17 @@ class FindAdDetailsForm extends UploadStep {
       key: _formKey,
       child: SingleChildScrollView(
         child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
           children: [
-            FormTitle(l10n.findAdDescriptionTitle),
+            FormTitle(l10n.findAdTitle),
             _titleField,
             _descriptionField,
-            FormTitle(l10n.findAdPriceTitle),
             _priceField,
-            FormTitle(l10n.findAdDateTitle),
+            FormFieldTitle(l10n.findAdDateTitle),
             _dateRangePicker,
+            SizedBox(
+              height: 24,
+            )
```

Comment:
NIT: I think it nicer with `const SizedBox(height: 24.0)`
Or maybe use padding on the column instead?